components/create-pull-request-modal.tsx

```diff
@@ -76,26 +78,58 @@ export default function CreatePullRequestModal({
       const filteredRepos = 
         repositories.filter(repo => (repo.isFork && repo.nameWithOwner === `${user.login}/${activeRepo?.name}`) 
                                     || repo.nameWithOwner === activeRepo?.githubPath);
-
       
-
       return Promise.all(filteredRepos
         .map(async (repository) => ({ repository, branches:  await getRepositoryBranches(repository.nameWithOwner)})));
     })
     .then(reposWithBranches => reposWithBranches
-      .map(({ repository, branches }) => branches.map(branch => ({ 
-        value: `${repository.owner}:${branch}`, 
-        label: branch,
-        postIcon: <Badge 
-                    color={repository.isOrganization ? "white-10" : "primary-30"}
-                    label={repository.isOrganization ? t("misc.organization") : t("misc.fork")}
-                  />,
-        isSelected: !!selectedBranch && branch === selectedBranch
-      })))
+      .map(({ repository, branches }) => branches
+        .map(branch => { 
+          
+          const isDisabled = !!activeIssue.pullRequests
```

Comment:
can we rename this variable to `existingPullRequest`? it clears up what it actually is, and aids on the core readability